_docs/_latest/setup/install.md

```diff
@@ -24,11 +24,189 @@ organization Cloud IAM policy.
+
+```bash
+terraform apply
+```
+
+Review the Terraform plan and enter `yes` to perform these actions.
+
+### Cleanup
+Remember to cleanup the service account used to install Forseti either manually or by running the command:
+
+```bash
+./scripts/cleanup.sh -p PROJECT_ID -o ORG_ID -s cloud-foundation-forseti-<suffix>
+```
+
+This will deprovision and delete the service account, and then delete the credentials file.
+
+If the service account was provisioned with the roles needed for the real time
+policy enforcer, you can set the `-e` flag to clean up those roles as well:
+
+```bash
+./scripts/cleanup.sh -p PROJECT_ID -o ORG_ID -S cloud-foundation-forseti-<suffix> -e
+```
+
+## Forseti Configuration
+Now that Forseti has been deployed, there are additional steps that you can follow to further 
+[configure Forseti]({% link _docs/latest/configure/index.md %}). Some of the commonly used features are 
+listed below:
+
+- [Enable G Suite Scanning]({% link _docs/latest/configure/inventory/gsuite.md %})
+- [Enable Cloud Security Command Center Notifications]({% link _docs/latest/configure/notifier/index.md %}#cloud-scc-notification)
+  - After activating this integration, add the Source ID into the Terraform configuration using 
+  the `cscc_source_id` input and re-run the Terraform `apply` command.
+
+## Requirements
+This section describes in detail the requirements necessary to deploy Forseti. The setup helper script automates the 
+service account creation and enabling the APIs for you. Read through this section if you are not using the setup script 
+or want to understand these details.
+
+1. Install Terraform.
+2. A GCP project to deploy Forseti into. The 
+[Google Project Factory Terraform](https://github.com/terraform-google-modules/terraform-google-project-factory) module 
+can be used to provision the project with the required APIs enabled, along with a Shared VPC connection.
+3. The Service Account used to execute this module has the right permissions.
+4. Enable the required GCP APIs to allow the Terraform module to deploy Forseti.
+
+## Software Dependencies
+
+### Terraform and Plugins
+- [Terraform](https://www.terraform.io/downloads.html) 0.12
+- [Terraform Provider for GCP](https://github.com/terraform-providers/terraform-provider-google) 2.11.0
+- [Terraform Provider Templates](https://www.terraform.io/docs/providers/template/index.html) 2.0
+
+## Service Account
+In order to execute this module you must have a Service Account with the following IAM roles assigned.
+
+### IAM Roles
+For this module to work, you need the following roles enabled on the Service Account.
+
+{% include docs/latest/forseti-terraform-sa-roles.md %}
+
+## Inputs
+Forseti provides many optional settings for users to customize for their environment and security requirements. 
+View the list of inputs below to see all of the available options.
+
+{% include docs/latest/forseti-terraform-inputs.md %}
```

Comment:
Do we want to include the TF inputs/outputs in the website? I think it is better to link to the TF repo here where we already have a script to update these inputs as new variables are changed. Since this list is changing a lot, I think it's better to keep that documented in one place in the TF repo. Same for the outputs.

Reply:
I agree that it's good to have the inputs/outputs in one place. @blueandgold do you want to jump in here?